### PR TITLE
ENH:toto-run:handle multiple products and materials

### PR DESCRIPTION
**Pull request by SantiagoTorres**
This fixes the argparse calls so that the --materials and --products
flags can recieve multiple paths as arguments without the need of quotes
or commas.

### CODE REVIEW

File: toto/toto-run.py [hunk 1]
```diff
@@ -93,8 +93,8 @@ def main():
   parser = argparse.ArgumentParser(
       description="Executes link command and records metadata",
       usage="toto-verify.py --step-name <unique step name>\n" +
```
Comment (lukpueh) [inline toto/toto-run.py]: Although it's not part of this PR. This should be `toto-run.py` and not `toto-verify.py`. Do you want to fix this here, or do we need an extra PR for this, or should we just hot push to develop?

File: toto/toto-run.py [hunk 2]
```diff
@@ -132,9 +132,9 @@ def main():
   product_list = []
 
   if args.materials:
-    material_list = args.materials.split(",")
+    material_list = args.materials
   if args.products:
-    product_list = args.products.split(",")
+    product_list = args.products
```
Comment (lukpueh) [inline toto/toto-run.py]: Any reason not to add `args.materials` and `args.products` directly to the function call in [`in_toto_run`](https://github.com/in-toto/in-toto/pull/32/files#diff-8e16d631a0ef00bfa92de5d2ad6c1f65R139)?

**Review by lukpueh — CHANGES_REQUESTED** [review]

File: toto/toto-run.py [hunk 3]
```diff
@@ -108,9 +108,9 @@ def main():
 
   # XXX LP: We should allow path wildcards here and sanitze them
```
Comment (lukpueh) [inline toto/toto-run.py]: I guess you can delete this comment now.

File: toto/toto-run.py (hunk 2 above)
Comment (SantiagoTorres) [reply toto/toto-run.py]: Not at all, I wanted to not pollute the diff (so that if we need to add any validation to it we donn't have problems). I could do so if you think this is better.

File: toto/toto-run.py (hunk 3 above)
Comment (SantiagoTorres) [reply toto/toto-run.py]: Will do.

File: toto/toto-run.py (hunk 2 above)
Comment (lukpueh) [reply toto/toto-run.py]: Hm. I'm fine with you polluting the diff.

**Comment by SantiagoTorres** [comment]
@lukpueh Ready for another round 👍

**Review by lukpueh — APPROVED** [review]
